modules/cmpapi/src/CallResponder.ts

```diff
@@ -1,9 +1,9 @@
-import {CommandCallback, TCFCommand} from './command';
-import {CommandMap} from './command/CommandMap';
-import {CmpApiModel} from './CmpApiModel';
-import {Disabled} from './response/Disabled';
-import {CustomCommands} from './CustomCommands';
-import {SupportedVersions} from './SupportedVersions';
+import {CommandCallback, TCFCommand} from './command/index.js';
```

Comment:
Wait so hold up, lemme understand this properly.  So if this library were just outputting ESM the `.js` extensions when importing/exporting would be unnecessary?  But since we are trying output both ESM and CJS, it is needed?

I just need more clarification because running through typescript files and seeing imports that contain `.js` file type is pretty confusing.

Reply:
Hi, the '.js' extension is required although this library would only produce in ESM.
(This is required because ESM also requires a full path with extension)

As you can read here (https://github.com/microsoft/TypeScript/issues/16577#issuecomment-754941937), there is no way to add '.js' via any Typescript configuration, this is because Typescript doesn't rewrite any part of the JavaScript code and this also implies the non-rewriting of the "module specifier" which in the case of using ESM must contain the suffix '.js'.

For this, as suggested by those who work and manage the Typescript project, I added the '.js' extension.

Let me know ;)

Reply:
Alright that sounds good.  I'll merge this in next Monday and publish a new pre-release so I can test it in my environment too.

Reply:
Perfect 👍